pkg/clusteragent/admission/mutate/common_test.go

```diff
@@ -203,128 +203,6 @@ func Test_shouldInject(t *testing.T) {
 			setupConfig: func() { mockConfig.SetWithoutSource("admission_controller.mutate_unlabelled", false) },
 			want:        false,
 		},
-		{
```

Comment:
Note for reviewers: all these tests only apply to auto instrumentation so I moved them to `pkg/clusteragent/admission/mutate/auto_instrumentation_test.go`